benchmark.cc

```diff
@@ -144,45 +144,32 @@ void RunBenchmark(const std::string& filename,
     total_pixels += numpixels;
   }
 
-  std::vector<uint16_t> delta_frame(xsize * ysize);
-  fpvc::ExtractFrame(raw.data(), xsize, ysize, shift, big_endian,
-                     delta_frame.data());
+  uint16_t *delta_frame = reinterpret_cast<uint16_t*>(raw.data());
```

Comment:
This isn't portable due to endianness, I think it's best to ensure the encoding works correctly on any computer?

Reply:
Hi Lode,

This should be possible, because the Encoder creates a Frame from the delta frame, also using the big_endian flag. During Frame creation, each pixel is split into high and low byte, taking shift-offset and endianess into account, the uint16_t is never realy (arithmeticaly) used as one. Or so i think, maybe i missed something?

Reply:
I believe this should fix it.
Bests,
Simon

Reply:
Pretty complicated but I think so indeed. Thanks for updating